infrastructure/cli/commands/helm/history.ts

```diff
@@ -20,26 +20,40 @@ type Release = {
 }
 
 export async function handler({environment}: EnvironmentOptions): Promise<void> {
+  const rows = (await history(environment))
+    .map(({revision, updated, status, appVersion, description}) => {
+      const updatedDate = shortDateTime(updated)
+      return {revision: String(revision), updated: updatedDate, status, appVersion, description}
+    })
+  printTable(rows, [
+    {name: 'Updated', key: 'updated'},
+    {name: 'Revision', key: 'revision'},
+    {name: 'Status', key: 'status'},
+    {name: 'Version', key: 'appVersion'},
+    {name: 'Description', key: 'description'},
+  ])
+}
+
+type History = {
```

Comment:
So this is release minus chart?

Does `appVersion` has a different case? I see the `Release` type has `app_version`, is this a typo thing or is this a helm thing?

Reply:
i'm forced to use "app_version" in `Release` cos that's what the helm command returns, but the JS norm is to camel case so that's what i use in _my_ type cos i have the control over it